src/index.js

```diff
@@ -261,33 +265,35 @@ const InputMask = forwardRef(function InputMask(props, forwardedRef) {
     setInputState(newInputState);
   });
 
+  const refCallback = node => {
+    inputRef.current = node;
+
+    // if a ref callback is passed to InputMask
+    if (isFunction(forwardedRef)) {
+      forwardedRef(node);
+    } else if (forwardedRef !== null && typeof forwardedRef === "object") {
+      forwardedRef.current = node;
+    }
+  };
+
   const inputProps = {
     ...restProps,
     onFocus,
     onBlur,
     onChange: isMasked && isEditable ? onChange : props.onChange,
     onMouseDown: isMasked && isEditable ? onMouseDown : props.onMouseDown,
-    ref: ref => {
-      inputRef.current = findDOMNode(ref);
-
-      if (isFunction(forwardedRef)) {
-        forwardedRef(ref);
-      } else if (forwardedRef !== null && typeof forwardedRef === "object") {
-        forwardedRef.current = ref;
-      }
-    },
     value: isMasked && isControlled ? lastValue : props.value
   };
 
   if (children) {
     validateChildren(props, children);
 
-    // We wrap children into a class component to be able to find
-    // their input element using findDOMNode
-    return <ChildrenWrapper {...inputProps}>{children}</ChildrenWrapper>;
+    // {@link https://stackoverflow.com/q/63149840/327074}
+    const onlyChild = React.Children.only(children);
+    return React.cloneElement(onlyChild, { ...inputProps, ref: refCallback });
```

Comment:
These lines were what was required to get the `children` working. As far as I can tell the main difference is that `React.Children.only(children)` returns the actual child which is then cloned where as `React.cloneElement(children)` which is what was being done in `<ChildrenWrapper>` returns a parent that you can't properly attach a `ref` to.